tests/screenshot/apps/components/TimePicker.js

```diff
@@ -20,6 +20,27 @@ const TimePickerTests = [
 		locale: 'ta-IN',
 		component: <TimePicker defaultValue={new Date(2009, 5, 6)} />
 	},
+	// start of [QWTC-2102]
+	{
+		locale: 'es-ES',
+		component: <TimePicker defaultValue={new Date(2009, 5, 6)} />
+	},
+	...withConfig({locale: 'ko-KR'}, [
+		<TimePicker defaultValue={new Date(2009, 5, 7, 11)} />,
+		<TimePicker defaultValue={new Date(2009, 5, 7, 13)} />
+	]),
+	...withConfig({locale: 'zh-HK'}, [
+		<TimePicker defaultValue={new Date(2009, 5, 7, 11)} />,
+		<TimePicker defaultValue={new Date(2009, 5, 7, 13)} />
+	]),
+	...withConfig({locale: 'am-ET'}, [
+		<TimePicker defaultValue={new Date(2009, 5, 7, 9)}>AM</TimePicker>,
+		<TimePicker defaultValue={new Date(2009, 5, 7, 12)}>Noon</TimePicker>,
+		<TimePicker defaultValue={new Date(2009, 5, 7, 13)}>PM</TimePicker>,
+		<TimePicker defaultValue={new Date(2009, 5, 7, 18)}>Evening</TimePicker>,
+		<TimePicker defaultValue={new Date(2009, 5, 7, 24)}>Late Evening</TimePicker>
```

Comment:
Do we need children prop for these Timepicker? Could you please check it?

Reply:
In Amharic, the time notation is different from the standard. Even though we input standard time, test result is different.
e.g)Standard noon: 12:00, Amharic noon: 06:00
It is to check the test result difference in Amharic.

Reply:
Okay.